src/alasqlparser.jison

```diff
@@ -589,6 +609,21 @@ ArrowDot
 	| DOT
 	;
 
+IntervalLiteral
+	: MICROSECOND
+	| MILLISECOND
+	| SECOND
+	| MINUTE
+	| HOUR
+	| DAY
+	| DAYOFWEEK
+	| DAYOFYEAR

```

Comment:
I dont think this is a valid interval. Can you please confirm?

Reply:
Correct, i will remove it.
The correct ones:

```
	year: 1000 * 3600 * 24 * 365,
	quarter: (1000 * 3600 * 24 * 365) / 4,
	month: 1000 * 3600 * 24 * 30,
	week: 1000 * 3600 * 24 * 7,
	day: 1000 * 3600 * 24,
	dayofyear: 1000 * 3600 * 24,
	weekday: 1000 * 3600 * 24,
	hour: 1000 * 3600,
	minute: 1000 * 60,
	second: 1000,
	millisecond: 1,
	microsecond: 0.001,
```

Reply:
I should have been more precise. 

I cant find documentation that `dayofyear` and `dayofmonth` is a valid unit for things like `interval`. I have a feeling those two should not be part of interval literals.

Reply:
I agree on that. If this will be part of a braking change release, this would be the moment to remove them from the implementation as well (and from the parser).

Reply:
For MySQL and MariaDB
   - `DAYOFYEAR`: Returns the day of the year (1-366).
   - `DAYOFMONTH`: Returns the day of the month (1-31).

So we will not remove it from the parser, but ill remove it from intervals.